instructure/canvas-android
# [Student][Teacher][MBL-14322] Html updates for newrce videos (and more)

Okay, so the first thing you'll notice is that this touches a lot of files. A pretty hefty amount of that is me refactoring a couple of our CanvasWebView html helper functions (unfortunately a lot of these are in the old parent app, so its just a method name refactor for most of those). We had two functions called formatHtml and a function called loadHtml. They were a sort of weird mismatch of each other, so I made it so we have just one loadHtml() and one formatHtml(). loadHtml() formats and loads, formatHtml() just formats and returns as a string.

Additionally, I combined a few of our helper functions for parsing out iframes, cnvs_content ids, and lti src urls into a single function, called loadHtmlWithiFrames. This has also been added to a decent number of locations with RCE content that didn't previously support it (discussions/quizzes/assignments in student being the biggest areas).

To Test:
There are a few old pages/etc with LTI content in the Android Dev course, I've been using those to check backwards compat with old rce content and lti embeds, all looks well there so far. I've also been using the new rce content to add videos in all of the other content types for testing. Let me know if you have any questions!
